### PR TITLE
Epoch loss should be averaged by sequence for validation and training

### PR DESCRIPTION
**Pull request by imccreadie**
Small fix to make the loss curves for training and validation be the average loss per sequence. Previously training was average loss per batch and validation was average loss per sequence.

### CODE REVIEW

**Review by dpacgopinath — APPROVED** [review]
Thanks for the fix!

**Comment by imccreadie** [comment]
FYI, I don't have write access so I can't merge this.

**Pull request merged**
